packages/core/medusa-test-utils/package.json

```diff
@@ -31,13 +31,17 @@
     "typescript": "^5.1.6"
   },
   "peerDependencies": {
+    "@medusajs/medusa": ">1.19",
```

Comment:
I got quite excited that we removed this :D Why do we need it back?

Reply:
Because in order to bootstrap the server we need some medusa loaders

Reply:
but it is optional as it is required only if you use the medusa integration test runner

Reply:
Actually, `medusa-test-runner.ts` file has a dynamic import on this one.